front/app/services/users.ts

```diff
@@ -126,7 +126,10 @@ export async function updateUser(userId: string, object: IUserUpdate) {
 
 export async function changePassword(object: IChangePassword) {
   const response = await streams.add<IUser>(`${apiEndpoint}/update_password`, {
-    user: object,
+    user: {
+      current_password: object.current_password,
+      new_password: object.password,
```

Comment:
Hmm, yeah is a bit odd that the backend accepts new_password, but returns errors for password. Easiest thing in future might be to change the backend so that the backend just uses password.